lib/components/kernel-monitor.js

```diff
@@ -66,22 +54,101 @@ const KernelMonitor = observer(({ store }: { store: store }) => {
       </ul>
     );
   }
-  const grouped = _.groupBy(
-    store.runningKernels,
-    kernel => kernel.gatewayName || "Local"
-  );
-  return (
-    <div className="kernel-monitor">
-      {_.map(grouped, (kernels, group) => (
-        <MonitorSection
-          store={store}
-          kernels={kernels}
-          group={group}
-          key={group}
-        />
-      ))}
-    </div>
-  );
+
+  const data = _.map(store.runningKernels, kernel => {
+    return {
+      gateway: kernel.gatewayName || "Local",
+      displayName: kernel.displayName,
+      status: kernel.executionState,
+      executionCount: kernel.executionCount,
+      kernel: kernel,
+      files: store.getFilesForKernel(kernel)
+    };
+  });
+  const columns = [
+    {
+      Header: "Gateway",
+      accessor: "gateway",
+      maxWidth: 150
+    },
+    {
+      Header: "Kernel",
+      accessor: "displayName",
+      maxWidth: 150
+    },
+    {
+      Header: "Status",
+      accessor: "status",
+      maxWidth: 100
+    },
+    {
+      Header: "Count",
+      accessor: "executionCount",
+      maxWidth: 75,
+      style: { textAlign: "right" }
+    },
+    {
+      Header: "Managements",
+      accessor: "kernel",
+      Cell: props => {
+        const kernel = props.value;
+        return [
+          <a
+            className="icon icon-playback-pause"
+            onClick={interrupt.bind(this, kernel)}
+            title="Interrupt kernel"
+            key={kernel.displayName + "int"}
+          />,
+          <a
+            className="icon icon-playback-rewind"
```

Comment:
What about something that looks like arrows pointing around, :recycle: . Maybe `icon-sync` for this?

Reply:
Ha, it's just because I couldn't find the icon when I once looked through the styleguide, but now I've finally found it and changed the restart icon to `icon-sync` !